microsoft/devhome
# User/dhoehna/fixing theming on content dialogs

## Summary of the pull request
This relates to content dialogs with content made from an adaptive card.
1. The adaptive card will not listen to the DevHome theme.  If Windows is in light mode and DevHome is in dark mode then the dialog box is in light mode while the adaptive card content is in dark mode.
2. The content dialog box will change themes when open.

This PR is a work around.  The work around is
1. Re-get the renderer.
3. Re-get the adaptive card
4. Re-assign the content of a dialog box to the adaptive car content retrieved earlier.

This issue only appears with content dialog boxes where the content is an adaptive card.

## References and relevant issues

https://task.ms/51611819

## Detailed description of the pull request / Additional comments

The only two instances I know of in DevHome are
1. Environments
2. QuickStart Playground.

Other instances will need to change over manually.

## Validation steps performed
Manual.

In addition to the two videos I tested changing the windows theme while DevHome's theme is set to Dark.  As expected the content dialog did not change theme.

## PR checklist

Movies!

Windows in light theme and DevHome in dark.
https://github.com/user-attachments/assets/60e48be1-dc5e-4d4a-a6b1-b9b3dde71573

Changing windows themes with DevHome theme set to "Windows Default"
https://github.com/user-attachments/assets/83327aa4-f5ad-489d-8650-4d822de749a7
